pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/VedleggOpplysningerOmEtteroppgjoeret.kt

```diff
@@ -411,102 +428,33 @@ data class OmBeregningAvBarnetillegg(
-        showIf(barnetillegg.totaltResultat.avvik.notEqualTo(Kroner(0))) {
-            paragraph {
-                textExpr(
-                    Bokmal to "Du har fått utbetalt ".expr() + barnetillegg.totaltResultat.fikk.format() + " kroner i barnetillegg. Du har fått " +
-                            barnetillegg.totaltResultat.avvik.absoluteValue().format() + " kroner for " +
-                            ifElse(barnetillegg.totaltResultat.harFaattForMye, "mye", "lite") + " i barnetillegg.",
-
-                    Nynorsk to "Du har fått utbetalt ".expr() + barnetillegg.totaltResultat.fikk.format() + " kroner i barnetillegg. Du har fått " +
-                            barnetillegg.totaltResultat.avvik.absoluteValue().format() + " kroner for " +
-                            ifElse(barnetillegg.totaltResultat.harFaattForMye, "mykje", "lite") + " i barnetillegg.",
-                    English to "You have received NOK ".expr() + barnetillegg.totaltResultat.fikk.format() + " in child supplement. You have been " +
-
-                            ifElse(barnetillegg.totaltResultat.harFaattForMye, "overpaid", "underpaid") + " NOK " +
-                            barnetillegg.totaltResultat.avvik.format() + " in child supplement.",
-                )
-            }
-        }
-
-        showIf(barnetillegg.totaltResultat.avvik.equalTo(Kroner(0))) {
+        val harFellesTillegg = barnetillegg.felles.notNull()
             paragraph {
-                textExpr(
-                    Bokmal to "Barnetillegget ditt har vært riktig beregnet ut fra inntekt i ".expr()
-                            + periode.format() + ",og utgjorde totalt " + barnetillegg.totaltResultat.fikk.format() + " kroner.",
-                    Nynorsk to "Barnetillegget ditt har vært riktig berekna ut frå inntekt i ".expr()
-                            + periode.format() + ",og utgjorde totalt " + barnetillegg.totaltResultat.fikk.format() + " koroner.",
-                    English to "Your child supplement has been correctly calculated in relation to your income in ".expr()
-                            + periode.format() + ", and totalled NOK " + barnetillegg.totaltResultat.fikk.format() + "."
-                )
-            }
-        }
-
-        title2 {
-            text(
-                Bokmal to "Beregningene er gjort hver for seg hvis:",
-                Nynorsk to "Utrekningane er gjort kvar for seg viss:",
-                English to "The calculations are done separately for each child if:"
-            )
-        }
-        paragraph {
-            list {
-                item {
-                    text(
-                        Bokmal to "du har flere barn som har ulike bosituasjoner.",
-                        Nynorsk to "du har fleire barn som har ulike busituasjoner.",
-                        English to "you have multiple children with different living arrangements."
+                showIf(
+                    pensjonsgivendeInntekt.inntekt.inntekter.isNotEmpty()
```

Comment:
Kan du tilordne dette boolske uttrykket til en variabel med et beskrivende navn? F.eks:
```
val erFlereTabeller = pensjonsgivendeInntekt.inntekt.inntekter.isNotEmpty() and (pensjonsgivendeInntekt.fratrekk.fratrekk.isNotEmpty() or harFellesTillegg)
```

Reply:
Har laget boolean variable. Har rettet "pensjonsgivendeInntekt" til:
val erflereTabellerPensjonsgivendeInntekt = (barnetillegg.personinntekt.inntekt.inntekter.isNotEmpty()
                    and (barnetillegg.personinntekt.fratrekk.fratrekk.isNotEmpty() or harFellesTillegg))